### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#196] [Q&A] [answered]
What are Solana clusters and what types exist?

**Comment by Tvenus** [comment, marked as answer]
Solana clusters are groups of validators that work together to maintain a single, consistent ledger. There are three main types of clusters:

Mainnet Beta: The live cluster where real SOL tokens are used.
Testnet: A cluster for testing purposes, using test SOL tokens.
Devnet: A developer cluster for testing applications without real economic impact.
